src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/paragraph/index.js

```diff
@@ -1,9 +1,16 @@
-import candyXmlToRichText from '#legacy/psammead-rich-text-transforms/src/candy-xml';
-import stringToRichText from '#legacy/psammead-rich-text-transforms/src/string';
-
+import loadable from '@loadable/component';
 import { processBlock } from './helpers';
 
+const richTextTransforms = loadable(() =>
+  import(
+    /* webpackChunkName: 'rich-text-transforms' */ '../../../../../../legacy/psammead-rich-text-transforms/src'
+  ),
+);
```

Comment:
Is there any reason this has been made to be `loadable` again?

As I understand it, we worked out during the psammead migration that we don't need this to be loadable as it will be included in the CPS story page bundle and not in other page type bundles.

This was orginally made loadable before we had page type bundle splitting so that it would only be downloaded on Story pages; with page type bundle splitting this has been made redundant.

Please check the above I may be wrong!

Reply:
It was done when we were testing the differences between latest and the migration stuff, it was an attempt to mirror latest's behaviour for bundling and the same moment version to double check how bundles were being laid out. I don't think this is necessary anymore and can be removed.